assets/blocks/course-progress-block/block.json

```diff
@@ -42,8 +42,7 @@
       "type": "integer"
     },
     "isPreview": {
-      "type": "boolean",
-      "default": false
```

Comment:
The default was removed because of an issue with the `example` attributes not working on patterns preview when it has a default value.

Reply:
I was debugging it a little more to open an issue in Gutenberg, and I just figured out that in the patterns inserter, it's displayed how it will be displayed in the editor.

<img width="356" alt="Screenshot 2023-11-21 at 17 55 17" src="https://github.com/Automattic/sensei/assets/876340/8096d174-8cee-4ada-9d41-5efbca0ba38d">

I checked some other blocks, and it doesn't use the `"example"`. Check my test adding an image to the pattern:

<img width="377" alt="Screenshot 2023-11-21 at 17 52 21" src="https://github.com/Automattic/sensei/assets/876340/bb35f181-4825-4a9c-b68e-788af9b25e6e">

The image uses the placeholder (that green rectangle on the top of the thumbnail) that is displayed in the editor. It doesn't show the video or image buttons, because in the editor, it's displayed only when the block is focused.